openslides_backend/services/datastore/adapter.py

```diff
@@ -199,14 +199,9 @@ def filter(
         get_deleted_models: DeletedModelsBehaviour = None,
```

Comment:
I would prefer to set the get_delete_models-parameter to a valid value corresponding it's type, i.e. DeletedModelsBehaviour.NO_DELETED, here and also in the following method-definitions

Reply:
ok, done.